packages/core/core.ts

```diff
@@ -90,10 +112,29 @@ export async function startFuzzingNoInit(
 	return Promise.resolve().then(() => fuzzerFn(fuzzFn, fuzzerOptions));
 }
 
-function stopFuzzing(err: unknown, expectedErrors: string[]) {
+function stopFuzzing(
+	err: unknown,
+	expectedErrors: string[],
+	coverageDirectory: string,
+	coverageReporters: reports.ReportType[]
+) {
 	if (process.env.JAZZER_DEBUG) {
 		trackedHooks.categorizeUnknown(HookManager.hooks).print();
 	}
+	// Generate a coverage report in fuzzing mode (non-jest). The coverage report for our jest-runner is generated
+	// by jest internally (as long as '--coverage' is set).
+	if (global.__coverage__) {
+		const coverageMap = libCoverage.createCoverageMap({});
+		coverageMap.merge(global.__coverage__);
```

Comment:
I don't know the `istanbul-lib-coverage` library well enough, but merging coverage necessary here?  `createCoverageMap` accepts a coverage variable and I am wondering if we can pass `__coverage__` directly and avoid calling `merge`

Reply:
You're right, the global `__coverage__` is already what `createCoverageMap` expects. Changed.